cli/endpoints/online/managed/vnet/setup/modules/nsg.bicep

```diff
@@ -0,0 +1,126 @@
+// Creates a network security group preconfigured for use with Azure ML
+// To learn more, see https://docs.microsoft.com/en-us/azure/machine-learning/how-to-access-azureml-behind-firewall
+@description('Azure region of the deployment')
+param location string
+
+@description('Tags to add to the resources')
+param tags object
+
+@description('Name of the network security group')
+param nsgName string
+
+resource nsg 'Microsoft.Network/networkSecurityGroups@2020-07-01' = {
+  name: nsgName
+  location: location
+  tags: tags
+  properties: {
+    securityRules: [      
+      {
+        name: 'AzureMachineLearning'
```

Comment:
minor: Suffix the name with Inbound - AzureMachineLearningInbound

Reply:
fixed. thanks